frontend/js/components/HistoryTable/HistoryTableRow.component.js

```diff
@@ -0,0 +1,29 @@
+class HistoryTableRow extends HTMLElement {
+  constructor() {
+    super();
+}
+connectedCallback() {
```

Comment:
single space between the function blocks

Reply:
fixed.